src/test/a11y/a11y.ts

```diff
@@ -68,7 +68,7 @@ describe('Accessibility', () => {
     Object.values(pathsRegistry).forEach((path: RoutablePath) => {
       const excluded = excludedPaths.some(_ => _ === path)
       if (!excluded) {
-        check(path.uri)
+        check(path.evaluateUri({ externalId: '91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6' }))
```

Comment:
do we need it?

Reply:
I mean `externalId` is not used anywhere as we mock methods with no arguments description

Reply:
Used in one view where nextUrl is validated.